services/json-rpc/cache/json_rpc_cache.go

```diff
@@ -17,6 +17,17 @@ import (
 
 const (
 	ChainIDHeader = "X-Forta-Chain-ID"
+
+	// These values will be injected into the agent container to configure SDK
+	// SDKRequestTimeout timeout until the SDK must fallback to the RPC Node
+	// Value in seconds and can be a float.
+	SDKRequestTimeout = "5"
```

Comment:
I would rename this to `SDKRequestTimeoutSeconds` and make it 20 if the chain data bucket interval is 10s. Also I think I would exclude the word `SDK` from any naming - the bot could be based on anything. Maybe `BotCacheRequestTimeout` kind of naming could be used.